<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registerscreen</class>
 <widget class="QMainWindow" name="Registerscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background: url(:/ui/main_bg.jpeg)  no-repeat center center fixed;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>0</y>
      <width>650</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/register your face (2).png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Enter_details">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
font: 70 24pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string>Enter Your Details</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Number">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 21pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Enrollment No.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 21pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_input">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_input">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Output">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 70 19pt &quot;Tahoma&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Take_image">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(170, 255, 255);
	font: 75 18pt &quot;Times New Roman&quot;;
	border-radius: 12px;
}
QPushButton:hover
{font: 75 20pt &quot;Times New Roman&quot;;
	background-color: rgb(75, 225, 225);
}
QPushButton:pressed 
{
	background-color: rgb(52, 143, 255);
}</string>
    </property>
    <property name="text">
     <string>Take Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Train_image">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(170, 255, 255);
	font: 75 18pt &quot;Times New Roman&quot;;
	border-radius: 12px;
}
QPushButton:hover
{font: 75 20pt &quot;Times New Roman&quot;;
	background-color: rgb(75, 225, 225);
}
QPushButton:pressed 
{
	background-color: rgb(52, 143, 255);
}</string>
    </property>
    <property name="text">
     <string>Train Image</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="registerscreen.qrc"/>
 </resources>
 <connections/>
</ui>
